packages/client-core/i18n/en/editor.json

```diff
@@ -622,7 +623,8 @@
         "cycle": "Burst Cycle",
         "interval": "Burst Interval",
         "probability": "Burst Probability"
-      }
+      },
+      "addBehavior": "Add Behavior"
```

Comment:
i believe this should be under the particle-system dict

Reply:
it should already be under it

```    "particle-system": {
      "name": "Particle System",
      "description": "Particle emitter to create particles.",
      "looping": "Looping",
      "duration": "Duration",
      "prewarm": "Prewarm",
      "emitter-shape": "Emitter Shape",
      "emission-bursts": "Emission Bursts",
      "shape-mesh": "Emitter Shape Mesh",
      "start-life": "Start Life",
      "start-size": "Start Size",
      "start-speed": "Start Speed",
      "start-rotation": "Start Rotation",
      "start-color": "Start Color",
      "emission-over-time": "Emission Over Time",
      "render-mode": "Render Mode",
      "texture": "Particle Texture",
      "u-tiles": "U Tiles",
      "v-tiles": "V Tiles",
      "start-tile-index": "Starting Tile Index",
      "mesh": "Particle Mesh",
      "blending": "Blending",
      "transparent": "Transparent",
      "world-space": "Use World Space",
      "burst": {
        "time": "Burst Time",
        "count": "Burst Count",
        "cycle": "Burst Cycle",
        "interval": "Burst Interval",
        "probability": "Burst Probability"
      },
      "addBehavior": "Add Behavior"
    },```

Reply:
it is, i swear it looked like it was under model, nonetheless